<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StabilogramVisualWidget</class>
 <widget class="QWidget" name="StabilogramVisualWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QFrame" name="frGraph">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="AreaGraph" name="wgtGraph" native="true">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollBar" name="sbSignal">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frToolbar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QFrame" name="frScale">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="lblScale">
           <property name="text">
            <string>Масштаб</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbScale">
           <property name="maxVisibleItems">
            <number>14</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnFullSignal">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../kernel.qrc">
          <normaloff>:/images/Tools/FullSignal.png</normaloff>:/images/Tools/FullSignal.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frFullSignal">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnHScalePlus">
           <property name="maximumSize">
            <size>
             <width>13</width>
             <height>13</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../../kernel.qrc">
             <normaloff>:/images/Tools/Plus_10.png</normaloff>:/images/Tools/Plus_10.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>9</width>
             <height>9</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnHScaleMinus">
           <property name="maximumSize">
            <size>
             <width>13</width>
             <height>13</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../../kernel.qrc">
             <normaloff>:/images/Tools/Minus_10.png</normaloff>:/images/Tools/Minus_10.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>10</width>
             <height>10</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnZeroing">
        <property name="toolTip">
         <string>Центровка</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../kernel.qrc">
          <normaloff>:/images/Animate/Zeroing.png</normaloff>:/images/Animate/Zeroing.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AreaGraph</class>
   <extends>QWidget</extends>
   <header location="global">areagraph.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../kernel.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cbScale</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>StabilogramVisualWidget</receiver>
   <slot>scaleChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>677</y>
    </hint>
    <hint type="destinationlabel">
     <x>956</x>
     <y>518</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnFullSignal</sender>
   <signal>clicked(bool)</signal>
   <receiver>StabilogramVisualWidget</receiver>
   <slot>btnFulSignalClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>660</y>
    </hint>
    <hint type="destinationlabel">
     <x>956</x>
     <y>546</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnZeroing</sender>
   <signal>clicked(bool)</signal>
   <receiver>StabilogramVisualWidget</receiver>
   <slot>btnZeriongClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>666</y>
    </hint>
    <hint type="destinationlabel">
     <x>958</x>
     <y>572</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnHScalePlus</sender>
   <signal>clicked()</signal>
   <receiver>StabilogramVisualWidget</receiver>
   <slot>btnPlusClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>657</y>
    </hint>
    <hint type="destinationlabel">
     <x>957</x>
     <y>505</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnHScaleMinus</sender>
   <signal>clicked()</signal>
   <receiver>StabilogramVisualWidget</receiver>
   <slot>btnMinusClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>673</y>
    </hint>
    <hint type="destinationlabel">
     <x>962</x>
     <y>603</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbSignal</sender>
   <signal>valueChanged(int)</signal>
   <receiver>StabilogramVisualWidget</receiver>
   <slot>signalScroll(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>634</y>
    </hint>
    <hint type="destinationlabel">
     <x>960</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>scaleChange(int)</slot>
  <slot>btnFulSignalClicked(bool)</slot>
  <slot>btnZeriongClicked(bool)</slot>
  <slot>btnPlusClicked()</slot>
  <slot>btnMinusClicked()</slot>
  <slot>signalScroll(int)</slot>
 </slots>
</ui>
